hub/static/js/explore.esm.js

```diff
@@ -493,7 +503,7 @@ const app = createApp({
         case 'filter':
           return dataset.is_filterable
         case 'shader':
-          return dataset.is_shadable
+          return ["party", "constituency_ruc"].includes(dataset.name) || !["text", "json", "date", "profile_id"].includes(dataset.data_type)
```

Comment:
Why hard-code these here, rather than relying on the `is_shadable` flag in the dataset model?